routes/v1.js

```diff
@@ -163,6 +181,58 @@ api.declare({
   });
 });
 
+
+/** List taskIds by taskGroupId */
+api.declare({
+  method: 'get',
+  route:  '/task-group/:taskGroupId/list',
+  query: {
+    continuationToken: /./,
+    limit: /^[0-9]+$/,
+  },
+  name:   'listTaskGroup',
+  output: 'list-task-group-response.json#',
+  title:  "List Task Group",
+  description: [
+    "List taskIds of all tasks sharing the same `taskGroupId`.",
+    "",
+    "As a task-group main contain an unbounded number of tasks, this end-point",
+    "may return a `continuationToken`. To continue listing tasks you must",
+    "`listTaskGroup` again with the `continuationToken` as the query-string",
+    "option `continuationToken`.",
+    "",
+    "By default this end-point will try to return up to 1000 members in one",
+    "request. But it **may return less**, even if more tasks are available.",
+    "It may also return a `continuationToken` even though there is no more",
+    "results. However, you can only be sure to have seen all results if you",
+    "keep calling `listTaskGroup` with a new `continationToken` until you",
+    "get a result without a `continuationToken`.",
+    "",
+    "If you're not interested in listing all the members at once, you may",
+    "use the query-string option `limit` to return fewer.",
+  ].join('\n')
+}, async function(req, res) {
+  let taskGroupId   = req.params.taskGroupId;
+  let continuation  = req.query.continuationToken || null;
+  let limit         = parseInt(req.query.limit || 1000);
+
+  let data = await this.TaskGroupMember.query({
+    taskGroupId,
+    expires: base.Entity.op.greaterThanOrEqual(new Date()),
```

Comment:
Is this putting a filter on "expires" that will only return records that have an expires greater than or equal to now?